### PR TITLE
fix(layout): SD-3658, SD-3948, SD-3965 Removed eye from package, layout issues in task, name in user unlock popup

### PR DESCRIPTION
**Pull request by pavlovicnemanja**
- [x] SD-3658 Delete eye button from package edit-> right menu
- [x] SD-3948 Layout issue in the tasks info section
- [x] SD-3965 The unlock popup window does not display the correct name of the user that unlocks the item

### CODE REVIEW

File: scripts/superdesk-packaging/packaging.js [hunk 1]
```diff
@@ -804,18 +804,6 @@
             backend: {rel: 'archived'}
         });
     }])
-    .config(['authoringWidgetsProvider', function(authoringWidgetsProvider) {
```
Comment (petrjasek) [inline scripts/superdesk-packaging/packaging.js]: I think it's used in some e2e tests, so this will require more changes

Comment (pavlovicnemanja) [inline scripts/superdesk-packaging/packaging.js]: Yep, I also deleted e2e test that was testing this widget.

**Comment by petrjasek** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**